src/pages/index.js

```diff
@@ -15,109 +12,8 @@ export default function Home() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <main className={styles.main}>
```

Comment:
@ibimina There are no styles defined in `Home.modules.css` so what are you importing here? 🤔

Reply:
I have removed the empty stylesheet import